Atomic/util.py

```diff
@@ -589,3 +591,54 @@ def expandvars(path, environ=None):
             i = len(path)
             path += tail
     return path
+
+def get_registry_configs(yaml_dir):
+    regs = {}
+    default_store = None
+    # Get list of files that end in .yaml and are in fact files
+    for yaml_file in [os.path.join(yaml_dir, x) for x in os.listdir(yaml_dir) if x.endswith('.yaml')
+            and os.path.isfile(os.path.join(yaml_dir, x))]:
+        with open(yaml_file, 'r') as conf_file:
+            try:
+                temp_conf = safe_load(conf_file)
+                if isinstance(temp_conf, dict):
+                    def_store = temp_conf.get('default-docker', None)
+                    if def_store is not None and default_store is not None:
+                        raise ValueError("There are duplicate entries for 'default-docker' in {}.".format(yaml_dir))
+                    elif default_store is None and def_store is not None:
+                        default_store = def_store
+                    registries = temp_conf.get('docker', None)
+                else:
```

Comment:
Since this has potential security concerns, I think we should probably error out.  But I think skopeo would fail on an atomic pull anyways.

The thing I would worry about would be allowing an image to be pull/run that did not pass the "trust" policy.